package.json

```diff
@@ -48,34 +48,45 @@
   "dependencies": {
     "@balena/jellyfish-assert": "^1.2.11",
     "@balena/jellyfish-logger": "4.0.27",
-    "@balena/jellyfish-plugin-base": "^2.2.25",
+    "@balena/jellyfish-mail": "^2.0.169",
+    "@balena/jellyfish-worker": "^16.1.3",
+    "bcrypt": "^5.0.1",
+    "blueimp-md5": "^2.19.0",
+    "googleapis": "^92.0.0",
+    "is-base64": "^1.1.0",
     "is-uuid": "^1.0.2",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@balena/jellyfish-action-library": "^16.2.0",
     "@balena/jellyfish-config": "^2.0.2",
     "@balena/jellyfish-core": "^14.1.0",
```

Comment:
This is now a main dependency

Reply:
Maybe I'm missing something, but can't we just drop this entirely since we're merging it into this repo in this PR?

Reply:
Sorry if it wasn't clear I meant `jellyfish-core`